channels/kw.m3u

```diff
@@ -1,51 +1,41 @@
+#EXTINF:-1 tvg-id="AlMaaref.kw" tvg-name="Al Maaref" tvg-country="KW" tvg-language="" tvg-logo="" group-title="",Al Maaref (350p)
+http://38.96.175.40:1935/liveTrans/ngrp:channel23_all/playlist.m3u8
 #EXTINF:-1 tvg-id="AlraiTV.kw" tvg-name="Alrai TV" tvg-country="KW" tvg-language="Arabic" tvg-logo="https://i.imgur.com/eqCEXk6.png" group-title="",Alrai TV (720p)
 https://media.streambrothers.com:1936/8724/8724/playlist.m3u8
-#EXTINF:-1 tvg-id="ATV.kw" tvg-name="ATV" tvg-country="KW" tvg-language="Arabic" tvg-logo="" group-title="",ATV (360p)
+#EXTINF:-1 tvg-id="ATV.kw" tvg-name="ATV" tvg-country="KW" tvg-language="Arabic" tvg-logo="http://www.atvkuwait.com/img/logo/atv_logo.png" group-title="",ATV (360p)
 https://gulfsat.cdn.easybroadcast.fr/live/Aladalah_abr/playlist.m3u8
 #EXTINF:-1 tvg-id="Ch4Teen.kw" tvg-name="Ch4Teen" tvg-country="KW" tvg-language="Arabic" tvg-logo="https://i.imgur.com/JeG9K1y.png" group-title="",Ch4Teen (480p)
 https://dcunilive93-lh.akamaihd.net/i/dclive_1@835787/master.m3u8
 #EXTINF:-1 tvg-id="Funoon.kw" tvg-name="Funoon" tvg-country="KW" tvg-language="Arabic" tvg-logo="" group-title="",Funoon (360p)
 https://gulfsat.cdn.easybroadcast.fr/live/FunoonHd_abr/playlist.m3u8
-#EXTINF:-1 tvg-id="KTV 1 ARB" tvg-name="KTV 1 ARB" tvg-country="KW" tvg-language="Arabic" tvg-logo="http://www.freeetv.com/images/03_logo/KTV1_Kuwait.jpg" group-title="",KTV 1 (1080p) [Offline]
+#EXTINF:-1 tvg-id="KTV1.kw" tvg-name="KTV 1" tvg-country="KW" tvg-language="Arabic" tvg-logo="http://www.freeetv.com/images/03_logo/KTV1_Kuwait.jpg" group-title="",KTV 1 (1080p) [Offline]
 https://hiplayer.hibridcdn.net/t/kwmedia-kwtv1.m3u8
-#EXTINF:-1 tvg-id="KTV 2 ARB" tvg-name="KTV 2 ARB" tvg-country="KW" tvg-language="Arabic" tvg-logo="http://www.freeetv.com/images/03_logo/ktv2_Kuwait.jpg" group-title="",KTV 2 (1080p) [Offline]
+#EXTINF:-1 tvg-id="KTV2.kw" tvg-name="KTV 2" tvg-country="KW" tvg-language="Arabic" tvg-logo="http://www.freeetv.com/images/03_logo/ktv2_Kuwait.jpg" group-title="",KTV 2 (1080p) [Offline]
 https://hiplayer.hibridcdn.net/t/kwmedia-kwtv2.m3u8
 #EXTINF:-1 tvg-id="KTVAlMajlis.kw" tvg-name="KTV Al Majlis" tvg-country="KW" tvg-language="Arabic" tvg-logo="http://i4.yhk0.net/cha/kw/almajlis-95x41.gif" group-title="",KTV Al Majlis (1080p) [Offline]
 https://hiplayer.hibridcdn.net/t/kwmedia-kwtvmajlis.m3u8
 #EXTINF:-1 tvg-id="KTVAlQurain.kw" tvg-name="KTV Al Qurain" tvg-country="KW" tvg-language="Arabic" tvg-logo="http://i5.satexpat.com/cha/kw/ktvQuran-95x88.gif" group-title="",KTV Al Qurain (1080p) [Offline]
 https://hiplayer.hibridcdn.net/t/kwmedia-kwtvqurain.m3u8
-#EXTINF:-1 tvg-id="KTV.kw" tvg-name="KTV" tvg-country="KW" tvg-language="Arabic" tvg-logo="http://content.osn.com/bob/745x419/KTA.jpg" group-title="",KTV Arabe (1080p) [Offline]
+#EXTINF:-1 tvg-id="KTVArabe.kw" tvg-name="KTV Arabe" tvg-country="KW" tvg-language="Arabic" tvg-logo="http://content.osn.com/bob/745x419/KTA.jpg" group-title="",KTV Arabe (1080p) [Offline]
```

Comment:
KTV channels are online, do you know if the format script is incorrectly setting the status? Thanks

Reply:
If the script finds a broken link, it always outputs the reason in the log: https://github.com/iptv-org/iptv/runs/3326687771?check_suite_focus=true#logs (tab "Format Playlist"). In this case it is "No streams found", that is, the playlist was loaded but there was no link to the stream. Most likely the broadcast was just interrupted during the test.

Reply:
Thanks